manifests/service/cors/configure.pp

```diff
@@ -81,9 +78,6 @@
         owner   => $owner,
         group   => $group,
         content => template('lightblue/properties/lightblue-cors.json.erb'),
-        before  => $before,
-        notify  => $notify,
-        require => $require,
```

Comment:
I don't know, but I asked the question over at http://ask.puppetlabs.com/question/15656/what-are-the-best-practices-for-passing-metaparameters-around/